doc/source/tutorial/signal.rst

```diff
@@ -905,86 +905,453 @@ in the amplitude response.
+     - :math:`\text{V}`
+     - :math:`\text{V}^2\text{s} / \text{Hz}`
+     - :math:`\text{V}^2 / \text{Hz}`
+     - :math:`\text{V} / \sqrt{\text{Hz}}`
+
+Note that the units presented in the table above are not unambiguous, e.g.,
+:math:`\text{V}^2\text{s} / \text{Hz} = \text{V}^2\text{s}^2 = \text{V}^2/
+\text{Hz}^2`. When using the absolute value of :math:`|X(f) / \tau|` of the amplitude
+spectrum, it is called magnitude spectrum. Furthermore, note that the naming scheme of
+the representations is not consistent and varies in literature.
+
+For real-valued signals the so-called "onesided" spectral is often utilized. It only
+uses the non-negative frequencies (due to :math:`X(-f)= \conj{X}(f)` if
+:math:`x(t)\in\IR`). Sometime the values of the negative frequencies are added to their
+positive counterparts. Then the amplitude spectrum allows to read off the full (not
+half) amplitude sine  of :math:`x(t)` at :math:`f_z` and the area of an interval in
+the PSD represents its full (not half) power. Note that for amplitude spectral
+densities the positive values are not doubled but multiplied by :math:`\sqrt{2}`, since
+it is the square root of the PSD. Furthermore, there is no canonical way for naming a
+doubled spectrum.
+
+The following plot shows three different spectral representations of four sine signals
+:math:`x(t)` of Eq. :math:numref:`eq_SpectA_sine` with different amplitudes durations
+:math:`a` and durations :math:`\tau`. For less clutter, the spectra are centered at
+:math:`f_z` and being are plotted next to each other:
+
+.. plot:: tutorial/examples/signal_SpectralAnalysis_ContinuousSpectralRepresentations.py
+
+Note that depending on the representation, the height of the peaks vary. Only the
+interpretation of the magnitude spectrum is straightforward: The peak at :math:`f_z` in
+the  represents half the magnitude :math:`|a|` of the sine signal. For all other
+representations the duration :math:`\tau` needs to be taken into account to extract
+information about the signal's amplitude.
+
+
+Sampled Sine Signal
+^^^^^^^^^^^^^^^^^^^
+In practice sampled signals are used. I.e., the signal is represented by :math:`n`
+samples :math:`x_k := x(kT)`, :math:`k=0, \ldots, n-1`, where :math:`T` is the sampling
+interval, :math:`\tau:=nT` the signal's duration and :math:`f_S := 1/T` the sampling
+frequency. Note that the continuous signal needs to be band-limited to :math:`[-f_S/2,
+f_S/2]` to avoid aliasing, with :math:`f_S/2` being called Nyquist frequency.
+[#Wiki_NyqistShannon]_  Replacing the integral by a sum to calculate the signal's
+energy and power, i.e.,
+
+.. math::
+
+    E_x = T\sum_{k=0}^{n-1} \abs{x_k}^2 = \frac{1}{2}|a|^2\tau\ , \qquad
+    P_x = \frac{1}{\tau}E_x =  \frac{1}{2}|a|^2\ ,
+
+delivers the identical result as in the continuous time case of Eq.
+:math:numref:`eq_SpectA_ContEnergy`. The discrete Fourier transform (DFT) and its
+inverse (as implemented in the :mod:`scipy.fft` module) is given by
```

Comment:
I think at some point it's useful to say/remind people somehow "FFT is just a specific implementation of the DFT" so could be hinted at here like,
```suggestion
inverse (as implemented using efficient FFT calculations in the :mod:`scipy.fft` module)
is given by
```